### ISSUE TITLE
First day of the week

### ISSUE THREAD

**Issue by nvllz** [0xf4b1/motionmate#79]
Hi, I would like the calendar week to start on Monday, but I don't see such a setting. Is there a simple way to implement this?

**Comment by 0xf4b1** [comment]
The first day of week is determined by the system. On Android 14, you can change it globally in settings at System > Languages > Regional preferences > First day of week and it will take effect system-wide also in apps like Google Calendar.

**Comment by nvllz** [comment]
Yeah, you're right. Unfortunately I'm on Android 13 and couldn't find this option. After switching device language to English (UK), the first day is set to Monday as expected. But date labels are still formated in MM/DD/YY.
